gapis/service/service.proto

```diff
@@ -1132,7 +1132,7 @@ message ProfilingData {
 
     // Entry contains performance data for a specific command.
     message Entry {
-      GpuSlices.Group group = 1;
+      int32 group = 1; // references slices.groups
```

Comment:
again should the name reflect what it refers instead of only commenting?

Reply:
Done.